keps/sig-network/2004-topology-aware-routing/kep.yaml

```diff
@@ -0,0 +1,49 @@
+title: Topology Aware Routing
+kep-number: 2004
+authors:
+  - "@ckyiricky"
+  - "@robscott"
+owning-sig: sig-network
+status: implementable
+creation-date: 2020-09-21
+reviewers:
+  - "@andrewsykim"
+  - "@bowei"
+  - "@freehan"
+  - "@thockin"
+approvers:
+  - "@thockin"
```

Comment:
Do we want another approver from another company, just for coverage?  @danwinship or @dcbw or other volunteer?